mona_sdk/mona_single_message.py

```diff
@@ -29,18 +32,32 @@ class MonaSingleMessage:
         message=<the relevant monitoring information>,
         contextClass="MY_CONTEXT_CLASS_NAME",
         contextId=<the context instance unique id>,
-        exportTimestamp=<the message export timestamp>,
+        sendTimestamp=<the message export timestamp>,
         action=<the wanted action>
     )
     """
 
+    def __post_init__(self):
```

Comment:
nit: Feels like this should be defined after the fields are define given that it uses them